Enterprise-CMCS/macpro-mdct-mfp
# MDCT-2832: Admin Banner Page

### Description

Adding the admin banner page for MFP.

**Note:** I am skipping the unit tests for `AdminBannerProvider` because these are dependent on the banner API calls, but they'll eventually be tested, trust.

### Related ticket(s)


MDCT-2832

---
### How to test

- Log in as `adminuser@test.com` and navigate to `/admin`

You should see a nice little banner form AND it has zero functionality! ✨ 

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
